pom.xml

```diff
@@ -863,7 +863,7 @@
         <carbon.kernel.registry.imp.pkg.version.range>[1.0.1, 2.0.0)</carbon.kernel.registry.imp.pkg.version.range>
 
         <!-- Carbon Identity Framework version -->
-        <carbon.identity.framework.version>5.25.305</carbon.identity.framework.version>
+        <carbon.identity.framework.version>5.25.346-SNAPSHOT</carbon.identity.framework.version>
```

Comment:
Is this a required change ?

Reply:
Just the one used locally. Need to update to the correct version when framework PR merged